RocketApp/Infrastructure/Sources/MotionClient/MotionClient.swift

```diff
@@ -0,0 +1,7 @@
+import Combine
+import ComposableCoreMotion
+import Foundation
+
+public struct MotionClient {
```

Comment:
Proc tu mas tohle? To uz nepotrebujes ne? Dyt registrujes naprimo ten `MotionManager`...

Reply:
Pravda. Smazu.